.travis.yml

```diff
@@ -23,6 +23,9 @@ python:
   - "3.6"
   - "3.5"
 script:
+  - python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); print(flags&os.O_NONBLOCK);'
```

Comment:
what's all this about?

Reply:
Trying to debug why travis-ci is failing, looks like it has something to do with printing out large blocks of text: https://github.com/travis-ci/travis-ci/issues/4704#issuecomment-348435959